lib/repo/repository.rb

```diff
@@ -360,9 +363,11 @@ def Repository.get_class(repo_type, conf_hash)
     # Check if configuration is in order
     config_keys.each do |c|
       if Repository.conf[c.to_sym].nil?
-        raise ConfigurationError.new("Required config '#{c}' not set")
+        raise ConfigurationError.new("get_class: " \
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.